src/gui/sharedialog.cpp

```diff
@@ -40,7 +40,7 @@ QString createRandomPassword()
     const auto words = OCC::WordList::getRandomWords(10);
 
     const auto addFirstLetter = [](const QString &current, const QString &next) {
-        return current + next.at(0);
+        return QString(current + next.at(0));
```

Comment:
I got bitten by QStringBuilder again I guess? :-)

I think in such a case I'd make the return value of the lambda explicit instead. But that's mainly a question of style at that point.